packages/opentelemetry-metrics/src/Handle.ts

```diff
@@ -50,7 +50,7 @@ export class CounterHandle implements types.CounterHandle {
    */
   getTimeSeries(timestamp: types.HrTime): TimeSeries {
     return {
-      labelValues: this._labelValues.map(value => ({ value })),
+      labelValues: Object.values(this._labels).map(value => ({ value })),
```

Comment:
Personally I feel the `TimeSeries` interface should accept `LabelSet` instead of just a list of strings (label values). There is no concept of implicit matching of ordering with keys and values if `LabelSet` is introduced, as it is not identified in the spec. Introducing a new ordering concept will just introduce confusion. Also, you should fix the docstrings for `TimeSeries` as well.